src/meep.hpp

```diff
@@ -42,6 +42,8 @@ typedef float realnum;
 typedef double realnum;
 #endif
 
+#define MIN_OUTPUT_TIME 4.0 // output no more often than this many seconds
```

Comment:
Should be renamed to `MEEP_MIN_OUTPUT_TIME` if it is in the global header.

Reply:
Fixed.